src/app/trade-creator-page/trade-creator-table/trade-creator-table.component.scss

```diff
@@ -0,0 +1,51 @@
+*, *:after, *:before {box-sizing: border-box;}
```

Comment:
we have stayed away from putting styling in these scss files located in directories. There could be a case made that it is nice to keep it separate, but generally diamondhands and redpartyhat have wanted all styling to be in the route styles.scss